python-searcher/python_searcher/searcher-cli.py

```diff
@@ -0,0 +1,187 @@
+import time
+from typing import List
+
+from solders.keypair import Keypair
+from solders.rpc.responses import GetBalanceResp
+
+from searcher import get_searcher_client
+from generated.searcher_pb2 import (
+    ConnectedLeadersRequest,
+    NextScheduledLeaderRequest,
+    PendingTxSubscriptionRequest,
+    NextScheduledLeaderResponse,
+)
+from generated.searcher_pb2_grpc import SearcherServiceStub
+import click
+
+from solders.transaction import VersionedTransaction
+from solana.rpc.api import Client
+
+
+@click.group("cli")
+@click.pass_context
+@click.option(
+    "--keypair-path",
+    help="Path to a keypair that is authenticated with the block engine.",
+    required=True,
+)
+@click.option(
+    "--block-engine-url",
+    help="Block Engine URL",
+    required=True,
+)
+def cli(
```

Comment:
what formatter you using? i like https://github.com/psf/black

Reply:
black